rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/41_search.yml

```diff
@@ -0,0 +1,337 @@
+---
+setup:
```

Comment:
This is an unfortunate level of duplication, and if a race condition affects any of these tests, I don't see how a dev would know to apply the fix to the duplicate.

Forgive me if these ideas have already been discussed and found unworkable, but if we can't find a syntax for expressing logical operations on features, maybe we should look at having some kind of directive for a code generator?